README.md

```diff
@@ -216,11 +216,8 @@ as a reference.
   Remove `ballast_size_mib` parameter from `memory_limiter` and make sure that it's added to
   `memory_ballast` extension as `size_mib` parameter instead:
 
-  ```yaml
-  extensions:
-    memory_ballast:
-      size_mib: ${SPLUNK_BALLAST_SIZE_MIB}
-  ```
+-  `memory_ballast` extension has been removed in favor of `GOMEMLIMIT` env var. If `GOMEMLIMIT` is not set, collector will enforce a soft limit automatically.
```

Comment:
Looks like this is being added to the upgrade guide's `### From 0.34.0 to 0.35.0` section. Were you trying to add it to some other section of the readme instead?

Reply:
Well, I thought we should give a message but this is redundant. I will remove it.